corehq/ex-submodules/casexml/apps/case/mock.py

```diff
@@ -99,7 +99,9 @@ def __init__(self,
         """
         super(CaseBlock, self).__init__()
         self._id = case_id
-        date_modified = date_modified or datetime.utcnow()
+        now = datetime.utcnow()
+        date_modified = date_modified or now
+        date_opened = date_opened or date_modified
```

Comment:
bumping my head against this now. I'm confused why you'd want to include 'date_opened' in the update block for a case that is already open?

I understand the need for it in the restore but this class is used for generating case blocks that go into forms.